insolar/record.go

```diff
@@ -59,6 +60,18 @@ func NewGlobalReference(local ID, base ID) *Reference {
 	return &global
 }
 
+// NewObjectReferenceFromBase58 deserializes reference from base58 encoded string and checks if it object reference
+func NewObjectReferenceFromBase58(input string) (*Reference, error) {
+	global, err := NewReferenceFromBase58(input)
+	if err != nil {
+		return nil, err
+	}
+	if !global.IsObjectReference() {
+		return nil, fmt.Errorf("provided reference is not object")
```

Comment:
errors.New() is better here, you don't need formatting

Reply:
Done!